engine/src/main/java/org/kigalisim/engine/number/UnitConverter.java

```diff
@@ -22,6 +22,13 @@
  * <p>This class handles unit conversions within the engine, supporting conversions
  * between volume units (kg, mt), population units (unit, units), consumption units
  * (tCO2e, kwh), time units (year, years), and percentage units (%).</p>
+ * 
+ * <p>Key differences from JavaScript implementation:
```

Comment:
Thank you for this! However, I think I will document this elsewhere. Please remove this new paragraph (<p>)